openssl_source/rand_lib.c

```diff
@@ -368,8 +372,6 @@ int RAND_bytes_ex(OSSL_LIB_CTX *ctx, unsigned char *buf, size_t num,
         return -1;
     }
 #endif
-    //use RDRAND for SGX Enclave insteadly
-    return get_sgx_rand_bytes(buf, num);
```

Comment:
Any specific consideration to remove this RDRAND call?

Reply:
I saw it has been moved to RAND_priv_bytes, the upper layer function.
It is a good question, if the user directly call RAND_bytes_ex, they may miss RDRAND?

Reply:
@lzha101  You are right. These lines should be kept.
@ShundaZhang And calling of `get_sgx_rand_bytes()` is also moved back to `RAND_bytes_ex()`, which can be called by `RAND_bytes()`.
Code is updated.